Add event for Mary Leung DiveRSE keynote (#753)
USRSE/usrse.github.io: _events/2022/2022-03-dei-diverse-leung.md

---
title: Normalizing inclusion by embracing difference
subtitle: DiveRSE Speaker Series
expires: 2022-03-23
event_date: "March 22, 2022"
layout: event

> duration: 60

Reviewer: ```suggestion
duration: 90
```

repeated: false
category: dei 
time:
    - - start: 2022-03-11T16:00:00Z
        end: 2022-03-11T17:30:00Z
---

## DiveRSE Speaker Series Info

> DiveRSE (Diverse RSE) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of EDI challenges and successes within the research software community.

Reviewer: ```suggestion
DiveRSE (Diverse RSE) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of DEI challenges and successes within the research software community.
```

Reviewer: Actually I think this is correct - either write it out here and add (EDI) or just write it out and define the acronym later. But we can't use the acronym without defining it first!

> DiveRSE (Diverse RSE) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of EDI challenges and successes within the research software community.

Reviewer: ```suggestion
[DiveRSE](https://diverse-rse.github.io/) (Diverse RSE) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of EDI challenges and successes within the research software community.
```

> [DiveRSE (Diverse RSE)](https://diverse-rse.github.io/) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of equite, diversity, and inclusion (EDI) challenges and successes within the research software community.

Reviewer: ```suggestion
[DiveRSE (Diverse RSE)](https://diverse-rse.github.io/) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of equity, diversity, and inclusion (EDI) challenges and successes within the research software community.
```

## Event - Normalizing inclusion by embracing difference

> DiveRSE is delighted to welcome Mary Ann Leung, Ph.D., Founder and President of the Sustainable Horizons Institute, as our inaugural speaker in the DiveRSE talk series looking at diversity, equity and inclusion in the Research Software Engineering (RSE) community.

Reviewer: ```suggestion
DiveRSE is delighted to welcome Mary Ann Leung, Ph.D., Founder and President of the [Sustainable Horizons Institute](https://shinstitute.org/), as our inaugural speaker in the DiveRSE talk series looking at diversity, equity and inclusion in the Research Software Engineering (RSE) community.
```

> DiveRSE is delighted to welcome Mary Ann Leung, Ph.D., Founder and President of the Sustainable Horizons Institute, as our inaugural speaker in the DiveRSE talk series looking at diversity, equity and inclusion in the Research Software Engineering (RSE) community.

Reviewer: I think there is a link for DiveRSE now?

Author: Yup! https://diverse-rse.github.io/

> The series runs from March 2022 and is linked to the [“Vive la différence - research software engineers” workshop](https://www.researchsoft.org/events/2022-04/) being hosted at the Lorentz Centre in The Netherlands in April 2022.

Reviewer: ```suggestion
The series runs from March 2022 and is linked to the ["Vive la différence - research software engineers" workshop](https://www.researchsoft.org/events/2022-04/) being hosted at the [Lorentz Center](https://www.lorentzcenter.nl/) in The Netherlands in April 2022.
```

> The keynote talk explores what it takes to achieve DEI, also known as equity, diversity, inclusion (EDI) and justice, equity, diversity, inclusion (JEDI) and how reframing DEI within RSE could increase innovation and developer productivity. We will also consider how cultivating respect and embracing difference could help to normalize inclusion.

Reviewer: JEDI! WICKED.

Following the talk, there will be a chance to join a facilitated discussion in breakout rooms providing an excellent opportunity to chat with Research Software Engineers and researchers from around the world.

This talk is being co-hosted with and supported by the [Society of Research Software Engineering](https://society-rse.org/).

More information about the event can be found at  [the event webpage](https://diverse-rse.github.io/events/2022-03-22).

[Register via Zoom](https://us06web.zoom.us/meeting/register/tZwvd-CprD4jH9Kt5smG4nGbp-YHoahn44Hm).
After registering, you will receive a confirmation email containing information about joining the meeting.

### Abstract

> The Vive la différence - research software engineers workshop aims to explore ways to reframe research software engineering (RSE) placing diversity, equity, and inclusion (DEI) as a central organizing principle. RSEs work in highly inter and multi-disciplinary environments, where they must be adept at “speaking” multiple languages, e.g., programming and domain languages. RSEs may have been trained in computer science, a domain science, mathematics, or an engineering discipline. Given the technical diversity, one might expect research software communities to welcome social diversity, creating an ideal ecosystem for the placing DEI as a central organizing principle. However, while research indicates that social diversity results in greater innovation, research software itself, as a growing multidisciplinary field spanning the globe and representing a diverse spectrum of technical domains, remains largely homogeneous.

Reviewer: ```suggestion
The Vive la différence - research software engineers workshop aims to explore ways to reframe research software engineering (RSE) placing diversity, equity, and inclusion (DEI) as a central organizing principle. RSEs work in highly inter and multi-disciplinary environments, where they must be adept at speaking multiple languages, e.g., programming and domain languages. RSEs may have been trained in computer science, a domain science, mathematics, or an engineering discipline. Given the technical diversity, one might expect research software communities to welcome social diversity, creating an ideal ecosystem for the placing DEI as a central organizing principle. However, while research indicates that social diversity results in greater innovation, research software itself, as a growing multidisciplinary field spanning the globe and representing a diverse spectrum of technical domains, remains largely homogeneous.
```
I'm an anti-put air quotes around things, feel free to disregard if you are an air quotes person.

Author: It's actually copy-pasta'd from the official website, so whoever wrote it clearly *is* an air quotes person.

Reviewer: lo

Reviewer: l

Reviewer: having trouble with typing today. Typing is hard.

Author: i
get
it
it's
pi
day

Reviewer: :pie: :pie: :pie: :pie: :pie: :pie: :pie:
